src/settings/DataImportSettings.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { Settings } from '@folio/stripes/smart-components';
+import { AppIcon, InfoPopover } from '@folio/stripes/components';
+
+import JobProfiles from './JobProfiles';
+import MatchProfiles from './MatchProfiles';
+import ActionProfiles from './ActionProfiles';
+import FieldMappingProfiles from './FieldMappingProfiles';
+import FileExtensions from './FileExtensions';
+
+import css from './DataImportSettings.css';
+
+class DataImportSettings extends Component {
+  sections = [
+    {
+      label: this.renderProfilesLabel(),
```

Comment:
i suggest to have routes as separated 'routes' files

Reply:
I think this is `Settings` component is just for it.